news/_posts/translation/do-you-speak-prestashop-january-2021.md

```diff
@@ -0,0 +1,116 @@
+ 
+Thank you for your involvement!
+
+## Top contributors
+ 
+A lot of you are working every day on Crowdin to have PrestaShop available in many languages, and PrestaShop cannot thank you enough for your dedication! Here are the 10 most active translators and proofreaders for January 2021.
+ 
+#### Top 10 translators in January:
+ 
+| |Translator | Language | # Words
+|-|---------- | -------- | ----------------
+| 1. | ‫דניאל שפירא‬‎ (danielshapiro9) | Hebrew | 4,264
+| 2. | mirmal | Esperanto | 3,598
+| 3. | jbonello | Maltese | 1,565
+| 4. | 8i11y | German | 989
+| 5. | Valerij Ivashchenko (likemusic) | Russian | 721
+| 6. | Armand (Rubujeto) | Esperanto | 655
+| 7. | Oleg (metacreo) | Russian, Lithuanian | 546
+| 8. | Tatu Wikman (tswfi) | Finnish | 324
+| 9. | A. L (roynsdx) | Arabic | 248
+| 10. | Çocuk Saatleri (cocuksaatleri) | Turkish | 218
+ 
+#### Top 10 proofreaders in January:
+ 
+| | Proofreader | Language | # Words
+|-| ---------- | -------- | ----------------
+| 1. | ‫דניאל שפירא‬‎ (danielshapiro9) | Hebrew | 4,346
+| 2. | A. L (roynsdx) | Arabic | 462
+| 3. | Rauno Riikman (weaver) | Finnish | 348
+| 4. | Monika (monikaraciunaite) | Lithuanian | 161
+| 5. | David Gonzalez (davidglezz) | Spanish | 149
+| 6. | Gabriel Tenita (ggedamed) | Romanian | 137
+| 7. | Arman Hosseini (Arman-Hosseini) | Persian | 102
+| 8. | webdvl (megashopba) | Czech | 56
+| 9. | Zoran Tejic (zotamal) | Serbian | 30
+| 10. | P.V. (MARK-APP) | Dutch | 21
+ 
+Congrats, and welcome to our new top contributors! :clap:
+ 
+Remember, you can see who has been contributing to our translation project thanks to the [Translators page](https://translators.prestashop.com/).
+ 
+## Complete translations
+ 
+#### Fully available languages
+ 
+At the end of January, PrestaShop 1.7.7 was fully available (= 100% translated and validated) in **14 languages**:
+ 
+| Bosnian | Chinese traditional | Danish | English |
+| French | Greek | Italian | Portuguese, Brazil |
+| **Romanian** | Serbian | Slovak | Slovene | 
+| **Spanish, Spain** | Swedish |
+
+## Languages with the best evolution
+
+The following languages had the best progress thanks to the translation community:
+ 
+* **Hebrew** (+8% to reach 95% translated) 
+* **Esperanto** (+8% to reach 69% translated)
+* **Russian** (+3% to reach 97% translated)
+
+Congratulations! :muscle:
+ 
+![Best translation progress for January 2021](/assets/images/2021/02/build-crowdin-progress-jan21.png)
+
+A huge thank you to all the contributors!
+ 
+Of course, this is highlighting the languages that made some progress with new translations; but it does not mean that the languages that are not mentioned here are not active. Indeed, some editing and rewriting could be going on, but the percentage of translation would not be modified (since it is working on strings that are already translated). So let's not forget about the work of other proofreaders! Thanks to you too!
+ 
+## Languages that need (more) proofreaders
```

Comment:
I'm wondering about this section... it has not changed for a while, should we turn it into a more relevant part while waiting for those projects to gain translations?

Reply:
I was thinking the same thing. Should we keep it like this for this time and change it next month?